src/directive/rateLimit.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  defaultKeyGenerator,
+  rateLimitDirective,
+} from 'graphql-rate-limit-directive';
+import { RateLimiterMemory } from 'rate-limiter-flexible';
+import { GraphQLError } from 'graphql';
+
+export let rateLimiter: CustomRateLimiterMemory;
+
+export class CustomRateLimiterMemory extends RateLimiterMemory {
```

Comment:
Please switch to use the Redis setup. We already have a running redis instance and an active connection so you can reuse it and I saw the package you use supports redis as well

Reply:
That would be best.